app/translations/cy/LC_MESSAGES/messages.po

```diff
@@ -889,7 +889,7 @@ msgstr "Dros y blynyddoedd diwethaf, rydym wedi profi'r holiadur llawn, gan gynn
 
 #: templates/static/accessibility.html:106
 msgid "<a href='{url}'>Digital Accessibility Centre (DAC)</a> carrying out a professional, technical audit"
-msgstr "●\tCael y <a href='{url}'>Ganolfan Hygyrchedd Digidol (DAC)</a> i gynnal archwiliad technegol proffesiynol"
+msgstr "cael y <a href='{url}'>Ganolfan Hygyrchedd Digidol (DAC)</a> i gynnal archwiliad technegol proffesiynol"
```

Comment:
Should `cael` be `Cael`?

Reply:
no, it's a bullet point and all other bullet points start with a lowercase character.

Reply:
:+1: